tests/test_calibrateImage.py

```diff
@@ -584,6 +586,65 @@ def test_calibrate_image_illumcorr(self):
         self.assertIn(key, result.exposure.metadata)
         self.assertEqual(result.exposure.metadata[key], True)
 
+    @staticmethod
+    def _mocked_requests_put(url, json=None):
```

Comment:
Same comments as in `ip_diffim`. If the Sattle access code is factored out, that should also eliminate most of the test redundancy.